docs/conf.py

```diff
@@ -258,6 +260,47 @@ def globalReplace(app, docname, source):
 
 global_replacements = {"{{PYODIDE_CDN_URL}}": CDN_URL}
 
+# Add extra CSS classes to some documentation fields. We use this in pyodide.css
+# to fix the rendering of autodoc return value annotations.
+from sphinx.util.docfields import Field
```

Comment:
Should we open an  issue about it at https://github.com/executablebooks/sphinx-book-theme ? It should ideally work with autodoc-typehints  without hacks.
Unless it's a problem with numpydoc that we are using

Reply:
I think this is more Napoleon's fault or at least it is fixing a problem that can be reproduced without autodoc-typehints.

Reply:
The issue specifically is that Napoleon (I think?) messes up the rendering if you don't provide an explicit return value type, either because you just don't want to, or because the return type annotation is filled in by some tool. It may also not be Napoleon's fault, there are so many different tools here that all only sort of work together that it is hard to tell where to place the blame when things don't work.